notes-docs.md

```diff
@@ -0,0 +1,51 @@
+Philosophy: one thing to do things
+
+Questions:
+
+-   What to do about the egghead lessons? They have a reference the decorator
+    syntax.
+
+-   The README is rather long. Credits, testimonials, and sponsors in multiple
+    places.
+
+-   How to name "compatible property tracking" as opposed to proxy-based?
+    How does one configure it, or it is an automatic fallback?
+
+-   What to do about translations? They will be out of date.
+
+-   Is there a way to regenerate assets? flow.png for instance uses
+    decorators and needs to be updated.
+
+-   Any code sandbox links need to be updated. jsfiddle as well.
+
+-   If 'observable' (and 'computed' and 'action') get another meaning,
+    does this mean that code that uses the old semantics has to import
+    these from mobx-decorators?
+
+-   Need to re-establish various links to the getting started sections in the
+    README.html.
+
+-   The example for React components in README.md won't work with the new strict default.
+    We need to either turn it off explicitly or introduce an action right away. I think
+    we should offer an action right away.
+
+-   When we introduce actions we need a link to an actions section that describes
+    how to turn off strict state reinforcing.
+
+-   What's the codemod all about?
+
+-   There are a lot of references to external documentation resources. They can probably use
+    a pruning.
```

Comment:
Feel free to do that if you find something like that. We can always revert if some turn out to be relevant.

Reply:
yeah, most of them are quite old, there are probably newer, better, and harder to find new ones:P